README.md

```diff
@@ -0,0 +1,15 @@
+# Kurtosis Starlark VsCode Extenstion (Official) (Pre-release)
+
+The kurtosis extension provides enriched experience for developers writing kurtosis packages in starlark. Some of the features this extension supports are:
+1. Syntax Highlighting for Starlark
+2. Autocomplete for Kurtosis defined instructions and types
+3. Hover functionality is available
```

Comment:
@Peeeekay is this for variables, types, and instructions?

```suggestion
3. Hover functionality
```

Reply:
for functions at the moment.